service-data-access/src/test/java/gov/va/vro/abd_data_access/service/FhirClientTest.java

```diff
@@ -4,10 +4,7 @@
 import ca.uhn.fhir.parser.IParser;
 import ca.uhn.fhir.rest.api.EncodingEnum;
 import gov.va.vro.abd_data_access.exception.AbdException;
-import gov.va.vro.abd_data_access.model.AbdBloodPressure;
-import gov.va.vro.abd_data_access.model.AbdClaim;
-import gov.va.vro.abd_data_access.model.AbdEvidence;
-import gov.va.vro.abd_data_access.model.AbdMedication;
+import gov.va.vro.abd_data_access.model.*;
 import lombok.extern.slf4j.Slf4j;
```

Comment:
Do not use *. That is one of the checks happen when you add spotless.

Reply:
Removed *.